src/sentry/static/sentry/app/views/organizationReleases/detail/commitAuthorStats.jsx

```diff
@@ -117,12 +117,7 @@ class CommitAuthorStats extends React.Component {
               return (
                 <PanelItem key={i} p={1} align="center">
                   <Flex>
-                    <Avatar
-                      user={author}
-                      size={20}
-                      hasTooltip
-                      tooltip={`${author.name} ${author.email}`}
-                    />
+                    <Avatar user={author} size={20} hasTooltip />
```

Comment:
@markstory I'm removing `tooltip` here since it wasn't being used or forwarded at all. I've fixed that in this PR.

In addition, I removed it overall since the fallback tooltip that uses `userDisplayName()` seems better.